DataFormats/SoATemplate/interface/SoALayout.h

```diff
@@ -0,0 +1,474 @@
+/*
```

Comment:
In `SoALayout.h` and `SoAView.h` the `\` at the end of the lines have gotten misaligned.

A suggestion for cleaning them up could be
  - temporarily remove the `clang-format off` comments
  - add `AlignEscapedNewlines: Right` to `.clang-format` if it's not there already
  - run `scram b code-format`
  - align any left-over stray `\` (like those inside comments)
  - add back the  `clang-format off` comments
  - undo the most annoying changes introduced by `clang-format`